_posts/nemothefish260/2022-12-03-Extra-Credit-Update-and-Rename-Clicky-Turtlehack.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+author: nemothefish260
+title: "CLicky Turtlehack"
+---
+
+**Link to my Clicky Turtlehack program**
+https://trinket.io/python/e636857dc0
```

Comment:
**IMPORTANT NOTE:**

The link in this post is not correct. It's showing incomplete code. Kindly refer to the code linked below:

https://trinket.io/python/ccd353c932